src/js/program/program-listeners.js

```diff
@@ -59,6 +59,9 @@ export function ProviderListener(mediaController) {
                 /* falls through to update duration while media is loaded */
             case MEDIA_TIME: {
                 mediaModel.set('position', data.position);
+                if (data.metadata && data.metadata.currentTime) {
+                    mediaModel.set('currentTime', data.metadata.currentTime);
+                }
```

Comment:
Is this already added to `event.metadata` in html5 and other providers? This should be moved to the event with these changes (`data.currentTime`).

Reply:
@robwalch 

I had a conversation with either @vseventer or @pajong about this. It was already in `data.metadata` and we didn't want to move it in case it broke something else. Thoughts?

Reply:
@waxidiotic Leave it there for backwards compatibility, but add it to the root of the event to meet the feature AC.

Reply:
@robwalch done